sdk/typespec/typespec_macros/README.md

```diff
@@ -0,0 +1,81 @@
+# TypeSpec Macros crate for Rust
+
+The TypeSpec Macros crate provides procedural macros for [TypeSpec](https://typespec.io)-generated client libraries. These macros simplify the implementation of common patterns required when working with TypeSpec-generated code.
+
+[Source code] | [Package (crates.io)] | [API reference documentation] | [TypeSpec documentation]
+
+## Getting started
+
+> **Note:** This crate is primarily intended for internal use by the `typespec_client_core` crate. Direct usage is discouraged unless you are contributing to or debugging the TypeSpec ecosystem.
+### Install the package
+
+> **Disclaimer:** The following instructions are intended for advanced users or contributors who need to work directly with the `typespec_macros` crate. Most users should depend on the `typespec_client_core` crate instead.
```

Comment:
That sort of info should go into `CONTRIBUTING.md`. That said, I see nothing below that really fits this sentence, so maybe just remove the sentence entirely.